### ISSUE TITLE
No Analysis for RTSP Stream 

### ISSUE THREAD

**Issue by philippneugebauer** [ultralytics/yolov5#3289]
Before submitting a bug report, please be aware that your issue **must be reproducible** with all of the following, otherwise it is non-actionable, and we can not help you:
 - **Current repo**: run `git fetch && git status -uno` to check and `git pull` to update repo
 - **Common dataset**: coco.yaml or coco128.yaml
 - **Common environment**: Colab, Google Cloud, or Docker image. See https://github.com/ultralytics/yolov5#environments
 
If this is a custom dataset/training question you **must include** your `train*.jpg`, `test*.jpg` and `results.png` figures, or we can not help you. You can generate these with `utils.plot_results()`.

## 🐛 Bug

I just want to analyze an RTSP stream but it just finishes analysis without doing anything. It is working with older versions.

## To Reproduce (REQUIRED)

Input:
```
python3 detect.py --source rtsp://host.docker.internal:8554/analysis
```

Output:
```
Fusing layers...
Model Summary: 224 layers, 7266973 parameters, 0 gradients
WARNING: Environment does not support cv2.imshow() or PIL Image.show() image displays
cv2.imshow() is disabled in Docker environments
1/1: rtsp://host.docker.internal:8554/analysis...  success (-3074457345618259 frames 1920x1080 at 30.00 FPS)

Results saved to runs/detect/exp9
Done. (0.000s)
```

## Expected behavior

Analysis of the stream

## Environment

I am using the yolo image from dockerhub with id 66b68c986fa6 on macos BigSur

**Comment by glenn-jocher** [comment]
@philippneugebauer good news 😃! Your original issue may now be fixed ✅ in PR #3295. To receive this update:
- **[Git](https://github.com/ultralytics/yolov5)** – `git pull` from within your `yolov5/` directory or `git clone https://github.com/ultralytics/yolov5` again
- **[PyTorch Hub](https://pytorch.org/hub/ultralytics_yolov5/)** – Force-reload with `model = torch.hub.load('ultralytics/yolov5', 'yolov5s', force_reload=True)`
- **[Notebooks](https://github.com/ultralytics/yolov5/blob/master/tutorial.ipynb)** – View updated notebooks  <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/yolov5)** – `sudo docker pull ultralytics/yolov5:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and informing us of the problem. Please let us know if this update resolves the issue for you, and feel free to inform us of any other **issues** you discover or **feature requests** that come to mind. Happy trainings with YOLOv5 🚀!

**Comment by philippneugebauer** [comment]
I can confirm that it works again :)
